packages/jaeger-ui/src/components/TracePage/TracePageHeader/AltViewOptions.tsx

```diff
@@ -58,8 +65,12 @@ export default function AltViewOptions(props: Props) {
   );
   return (
     <Dropdown overlay={menu}>
-      <Button className="ub-mr2" htmlType="button" onClick={onTraceGraphViewClicked}>
-        {traceGraphView ? 'Trace Graph' : 'Trace Timeline'} <Icon type="down" />
+      <Button
+        className="ub-mr2"
+        htmlType="button"
+        onClick={() => onTraceGraphViewClicked(selectedTraceView + 1)}
```

Comment:
> I suggest having this button should show the label of the current view and do nothing when the button itself is clicked.
+1
We had discussed doing this even before this PR. So rather than complicating the button I agree it should only be a dropdown and not a button.